dmutils/documents.py

```diff
@@ -190,6 +190,16 @@ def get_agreement_document_path(framework_slug, supplier_id, supplier_name, docu
     )
 
 
+def get_countersigned_agreement_document_path(framework_slug,
+                                              supplier_id,
+                                              document_name="countersigned-framework-agreement.pdf"):
```

Comment:
It feels like the wrong function name if we want the document name to be parameterised. It's a bit of a pain because `get_agreement_document_path` would be perfect but it's taken.